### ISSUE TITLE
Allow profiles to be based on all modeling languages

### ISSUE THREAD

**Issue by danyeaw** [gaphor/gaphor#1486]
### Describe the bug
```
Poe => raaml
WARNING:gaphor.storage.storage:file models/RAAML.gaphor could not be loaded (Type Block can not be loaded: no such element 25f62e28-c33b-11ec-819a-f47b099bf663)
Traceback (most recent call last):
  File "<string>", line 2, in <module>
  File "/home/dan/Projects/gaphor/gaphor/codegen/coder.py", line 75, in main
    model = load_model(modelfile)
  File "/home/dan/Projects/gaphor/gaphor/codegen/coder.py", line 408, in load_model
    storage.load(
  File "/home/dan/Projects/gaphor/gaphor/storage/storage.py", line 231, in load
    for status in load_generator(filename, factory, modeling_language):
  File "/home/dan/Projects/gaphor/gaphor/storage/storage.py", line 271, in load_generator
    for percentage in load_elements_generator(
  File "/home/dan/Projects/gaphor/gaphor/storage/storage.py", line 148, in load_elements_generator
    yield from _load_elements_and_canvasitems(
  File "/home/dan/Projects/gaphor/gaphor/storage/storage.py", line 192, in _load_elements_and_canvasitems
    create_element(elem)
  File "/home/dan/Projects/gaphor/gaphor/storage/storage.py", line 181, in create_element
    assert cls, f"Type {elem.type} can not be loaded: no such element {elem.id}"
AssertionError: Type Block can not be loaded: no such element 25f62e28-c33b-11ec-819a-f47b099bf663

```

### To Reproduce
1. Edited the RAAML.gaphor file.
2. Ran `poe raaml`
3. See error above, model file is attached.
[RAAML.gaphor.zip](https://github.com/gaphor/gaphor/files/8550173/RAAML.gaphor.zip)

### OS
- [X] Linux (Please put in notes the specific distro)
- [ ] macOS
- [ ] Windows

### Version
Version of Gaphor: main branch

**Comment by amolenaar** [comment]
On `main` `poe raaml` works, did you change/add something?

**Comment by danyeaw** [comment]
If I start adding Blocks that are Stereotyped with Situation, the models become corrupted. See the raaml.gaphor model attached to this issue.

**Comment by amolenaar** [comment]
Ah, the "problem" is in the function `load_model` (https://github.com/gaphor/gaphor/blob/main/gaphor/codegen/coder.py#L403).

The ModelingLanguage is using the Core and UML modeling language. The SysML modeling language is not present, hence `Block` can not be found.

It would be better if the coder will use the `gaphor.modelinglanguages` entry points instead.

**Comment by danyeaw** [comment]
@amolenaar Ahh yes, I forgot that we had based all the profiles on UML only to start. Nice find!

**Comment by amolenaar** [comment]
Only: why are you defining a Block, instead of a Class? For the (meta) model, shouldn't we just use classes. A class named "Block", if you like?

**Comment by danyeaw** [comment]
@amolenaar Because RAAML is really build on top of SysML. I don't think we would want to have to redefine things like ValueTypes and Blocks inside RAAML right?

**Comment by danyeaw** [comment]
Closed by #1494

**Comment by amolenaar** [comment]
> @amolenaar Because RAAML is really build on top of SysML. I don't think we would want to have to redefine things like ValueTypes and Blocks inside RAAML right?

That makes sense. It's best to stick as close to the original as possible.
